<launch>
    <node name="dynamixel_control" pkg="motors" type="dynamixel_control" output="screen" clear_params="true" ns="motors">
        <param name="port_adress" value="/dev/ttyUSB1" />

        <param name="init_position_1" type="int" value="2000" />
        <param name="init_position_2" type="int" value="2000" />
        <param name="init_position_3" type="int" value="2000" />
        <param name="init_position_4" type="int" value="2000" />

        <param name="init_offset_1" type="int" value="-1000" />
        <param name="init_offset_2" type="int" value="-400" />
        <param name="init_offset_3" type="int" value="700" />
        <param name="init_offset_4" type="int" value="-1000" />

        <param name="velocity_mode" type="bool" value="true" />

    </node>
</launch>
